docs/FastAPI.md

```diff
@@ -7,6 +7,26 @@
 * `AGENT_CONFIG` - REQUIRED - Path to the agent configuration file in yaml format.
   Check [the agent configurations here](./AgentConfig.md).
 
+### Redis
```

Comment:
I probably missed something, but why didn't these become another section in the agent's configuration YAML?

Reply:
I thought that the agent config was the config for the whole application, hence my misconception.